CONTRIBUTING.md

```diff
@@ -30,14 +30,21 @@ https://www.eclipse.org/projects/handbook/#resources-commit
 
 ## Setting up the Development Environment
 
-To build project you need to install npm.
+To build project you need to install npm: https://www.npmjs.com/
+
+Checkout source code of Explorer for z/OS reaching the repository at https://github.com/eclipse/che-che4z-explorer-for-zos address.
+* fork the project
+* then clone or download it
 
-Checkout source code of Explorer for z/OS:
 $ git clone https://github.com/eclipse/che-che4z-explorer-for-zos.git
 
-To build project run:
+Build the project running:
 $ npm install && npm run compile
 
+When you finished type the following:
+* $ git add .
+* $ git commit -m "explanation of your work" -s
```

Comment:
Issue number also has to be in a commit message.

Reply:
Thanks. I will describe it.

Reply:
... soon.

Reply:
Did other modification. Please let me know.

Reply:
https://accounts.eclipse.org/legal/eca/validation/12460

Reply:
@Gianni-Pucillo   please close the pull request if you have another one open for the same issue

Reply:
Yes I close it. Thanks